.wwhrd.yml

```
@@ -1,29 +1,29 @@
 ---
 blacklist:
-  - GPL-2.0
+  - GPL-2.0+
+  - LGPL-2.1+
+  - AFGPL-3.0+
 
 whitelist:
   - Apache-2.0
-  - FreeBSD
-  - LGPL-3.0
+  - BSD-2-Clause
   - MIT
   - MPL-2.0
-  - NewBSD
+  # Incompatible with GPL-2.0+, LGPL-2.1+, or AFGPL-3.0+
+  - MPL-2.0-no-copyleft-exception
+  - BSD-3-Clause
 
 exceptions:
   - bitbucket.org/ww/goautoneg # Unknown
-  - github.com/BurntSushi/toml # MIT
-  - github.com/davecgh/go-spew/spew # ISC
-  - github.com/howeyc/gopass # ISC
+  - github.com/davecgh/go-spew/spew # ISC, misdetected as 0BSD
+  - github.com/gogo/protobuf/protoc-gen-gogo/descriptor # BSD-3-Clause
+  - github.com/gogo/protobuf/gogoproto # BSD-3-Clause
+  - github.com/gogo/protobuf/proto # BSD-3-Clause
+  - github.com/gogo/protobuf/sortkeys # BSD-3-Clause

```

These 4 lines could ne replaced by `github.com/gogo/protobuf/...`
```suggestion
  - github.com/gogo/protobuf/... # BSD-3-Clause
```

---

Ah, thank you. I tried `github.com/gogo/protobuf/` initially, but didn't know about the `...` syntax. I'll fix that now.